Docs: add hidden_users configuration field (#30435)
grafana/grafana: docs/sources/whatsnew/whats-new-in-v7-4.md

You can lock down what can be done in the frontend code. Lock down what can be loaded, what JavaScript is executed. Not compatible with some plugins.

### Hide users in UI

> You can now use the `hidden_users` configuration setting to hide specific users in the UI. With this feature, users created for automation purpose can never be shown in Grafana UI.

Reviewer: ```suggestion
You can now use the `hidden_users` configuration setting to hide specific users in the UI. For example, this feature can be used to hide users that are used for automation purposes.
```

Perhaps?
